src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js

```diff
@@ -74,16 +98,20 @@ const CompanyOffersManagementWidget = () => {
         }).catch((err) => {
             setError(err[0]);
             setIsLoading(false);
+            addSnackbar({
+                message: "An unexpected error occurred, please try refreshing the browser window.",
+                key: `${Date.now()}-fetchOffersError`,
+            });
         });
-    }, [data.company._id, isLoggedIn]);
+    }, [addSnackbar, data.company._id, isLoggedIn]);
 
     const RowContent = ({ rowKey, labelId }) => {
         const fields = offers[rowKey].fields;
 
         return (
             <>
                 {Object.entries(fields).map(([fieldId, fieldOptions], i) => (
-                    generateTableCellFromField(i, fieldId, fieldOptions, labelId)
+                    GenerateTableCellFromField(i, fieldId, fieldOptions, labelId)
```

Comment:
Ahhh, now I get it... In that case `GenerateTableCellFromField` is indeed a component. We should rename it to `FieldTableCell` or something, make sure the props are received correctly (an object of props instead of spread arguments) and change the function to something like this:

```
const RowContent = ({ rowKey, labelId }) => {
        const fields = offers[rowKey].fields;

        return (
            <>
                {Object.entries(fields).map(([fieldId, fieldOptions], i) => (
                    <FieldTableCell id={i} fieldId={fieldId} fieldOptions={fieldOptions} labelId={labelId} />
                ))}
```